### ISSUE TITLE
Add more information about `plasmapy_sphinx` to doc guide

### ISSUE THREAD

**Issue by namurphy** [PlasmaPy/PlasmaPy#1194]
After #1156 is merged, it would be helpful to expand upon `plasmapy_sphinx` in the documentation guide in the development guide.  As of now, #1156 only briefly mentions `plasmapy_sphinx` and says that it is expected to be broken out into its own package in the future.

**Comment by rocco8773** [comment]
... I know we're going to break `plasmapy_sphinx` into it's own pkg.  I feel maybe we should delay the suggestion of this issue until we do the split, just so we don't have to comeback on a second round.  Of course that means you all need to give me a swift kick in the rear so I don't drag my feet. 🤣

**Comment by namurphy** [comment]
All good points!  #1156 is barely adding anything at all about `plasmapy_sphinx`, so the doc guide would still need at least a few more sentences about `plasmapy_sphinx` even after it becomes its own package.  But yeah, it seems totally reasonable to me to wait to address this issue until after the split.

**Comment by namurphy** [comment]
This will probably be superseded by #1643.

**Comment by namurphy** [comment]
Closing this issue due to plans to extract `plasmapy_sphinx` from `docs/` and make it its own package.
